05-onchain.md

```diff
@@ -148,9 +150,10 @@ A node:
         - otherwise:
           - MUST broadcast the *last commitment transaction*, for which it has a
           signature, to perform a *unilateral close*.
-          - MUST spend any `to_local_anchor` output, providing sufficient fees as incentive to include the commitment transaction in a block.
-          Special care must be taken when spending to a third-party, because this re-introduces the vulnerability that was
-          addressed by adding the CSV delay to the non-anchor outputs.
+          - MUST spend any `to_local_anchor` or `shared_anchor` output, providing sufficient fees
+          as incentive to include the commitment transaction in a block. When using `nVersion=2`,
+          special care must be taken when spending to a third-party, because this re-introduces
+          the vulnerability that was addressed by adding the CSV delay to the non-anchor outputs.
```

Comment:
Bringing up `nVersion=2` here is a bit confusing and may give the reader the idea they can set `nVersion=3` for `to_local_anchor` spends to avoid this issue (which won't relay).

```suggestion
          - MUST spend any `to_local_anchor` or `shared_anchor` output, providing sufficient fees
          as incentive to include the commitment transaction in a block. For `to_local_anchor`,
          special care must be taken when spending to a third-party, because this re-introduces
          the pinning vulnerability that was addressed by adding the CSV delay to the non-anchor outputs.
```